project/app/people.py

```diff
@@ -0,0 +1,16 @@
+class Person(object):
+    def __init__(self):
+        pass
+
+    def add_person():
+        pass
+
+
+class Fellow(Person):
+    def __init__(self):
+        pass
+
+
+class Staff(Person):
+    def __init__(self):
+        pass
```

Comment:
a function that passes is a function that is not needed.. we don't nee to call something that passes